.github/workflows/build.yml

```diff
@@ -327,7 +329,7 @@ jobs:
         echo " "
         make -f certifier.mak clean
         make -f certifier_tests.mak clean
-        ENABLE_SEV=1 make -f certifier.mak
+        ENABLE_SEV=1 make -j2 -f certifier.mak
```

Comment:
More such instances of ... Minor update to manual make in CI to use parallelism.